kratos/mpi/utilities/parallel_fill_communicator.cpp

```diff
@@ -208,7 +208,7 @@ void ParallelFillCommunicator::ComputeCommunicationPlan(ModelPart& rModelPart)
     // Get number of processors.
     const int num_processors = mrDataComm.Size();
     // Find all ghost nodes on this process and mark the corresponding neighbour process for communication.
-    vector<bool> receive_from_neighbour(num_processors, false);
+    std::vector<bool> receive_from_neighbour(num_processors, false);
```

Comment:
VERY careful with this. vector<bool> should be the same as DenseVector<bool>=boost::numeric::ublas::vector<bool>

std::vector<bool> is a different (and very weird) beast

Reply:
Do you remember what is expected to be here?, because anyway if it is supposed to be DenseVector I woulld switch to DenseVector instead. I have noticed that Ublas is polluting completelly our code and yhis would make future refactors and replacements very difficult.